### PR TITLE
Display message on product page when customer subscribes to stock notification

### PR DESCRIPTION
**Pull request by tonyyb**
Before this PR customer don't know on product page if he is sucribed to notification.

:warning: :warning:
- This PR includes breaking change if the theme overrides the `product.tpl` view.
- A new translation must be added to PrestaShop : `You will be notified when product becomes available.` with `Modules.Mailalerts.Shop` domain.

Other solution without breaking change : create a new view only for this usage.

### CODE REVIEW

**Comment by tonyyb** [comment]
What about this PR @Joukz ?

File: views/templates/hook/product.tpl [hunk 1]
```diff
@@ -22,20 +22,25 @@
...
-                {hook h='displayGDPRConsent' id_module=$id_module}
-            {/if}
-            <input type="hidden" value="{$id_product}"/>
-            <input type="hidden" value="{$id_product_attribute}"/>
-            <button class="btn btn-primary" type="submit" rel="nofollow" onclick="return addNotification();">{l s='Notify me when available' d='Modules.Mailalerts.Shop'}</button>
-            <span style="display:none;padding:5px"></span>
+    {if !$has_notification}
+        <form>
+            <div class="js-mailalert" style="text-align:center;" data-url="{url entity='module' name='ps_emailalerts' controller='actions' params=['process' => 'add']}">
+                {if isset($email) AND $email}
```
Comment (Progi1984) [inline views/templates/hook/product.tpl:29]:
```suggestion
                {if !empty($email)}
```

File: views/templates/hook/product.tpl [hunk 2]
```diff
@@ -22,20 +22,25 @@
...
+                {/if}
+                <input type="hidden" value="{$id_product}"/>
+                <input type="hidden" value="{$id_product_attribute}"/>
+                <button class="btn btn-primary" type="submit" rel="nofollow" onclick="return addNotification();">{l s='Notify me when available' d='Modules.Mailalerts.Shop'}</button>
+                <span style="display:none;padding:5px"></span>
+            </div>
+        </form>
+    {else}
+        <div class="alert alert-info">
+            {l s='You will be notified when product becomes available.' d='Modules.Mailalerts.Shop'}
```
Comment (LouiseBonnard) [inline views/templates/hook/product.tpl:43]:
```suggestion
            {l s='You will be notified when the product becomes available.' d='Modules.Mailalerts.Shop'}
```

Comment (LouiseBonnard) [reply views/templates/hook/product.tpl:43]: Following the work of PR #49. :-)

**Comment by matks** [comment]
Hi,

Since we had no news from you more than 6 months, I will follow [our guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/#stale-pull-requests) and I'll close this pull request as it seems stale. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Pull request closed without merging**